payments/schema/mutations.py

```diff
@@ -171,6 +176,76 @@ def mutate_and_get_payload(cls, root, info, **input):
         return DeleteWinterStorageProductMutation()
 
 
+class AdditionalProductInput:
+    service = ServiceTypeEnum()
+    period = PeriodTypeEnum()
+    price_value = graphene.Decimal()
+    price_unit = PriceUnitsEnum()
+    tax_percentage = AdditionalProductTaxEnum()
+
+
+class CreateAdditionalProductMutation(graphene.ClientIDMutation):
+    class Input(AdditionalProductInput):
+        service = ServiceTypeEnum(required=True)
+        period = PeriodTypeEnum(required=True)
+        price_value = graphene.Decimal(required=True)
+
+    additional_product = graphene.Field(AdditionalProductNode)
+
+    @classmethod
+    @add_permission_required(AdditionalProduct)
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, **input):
+        input["tax_percentage"] = AdditionalProductTaxEnum.get(
+            input.pop("tax_percentage")
+        ).value
```

Comment:
hm, tested this locally just now, seems to work fine without this, as `input` already has `Decimal` object

Reply:
i.e. graphene converts it from name to value internally

Reply:
Ohh good catch, once again Graphene outsmarts me.